python/lsst/ts/linearstage/controllers/igus_dryve.py

```diff
@@ -156,7 +167,7 @@ async def connect(self):
         # Now verify that controller is enabled (DI7 is high)
         # This must be done from the controller GUI or by sending a 24V
         # signal to the input.
-        response = await self.retrieve_status()
+        response: tuple = await self.retrieve_status()
 
         # Check that byte 20 is has bits 0,1,2 set
         if not (response[20] & 0b111):
```

Comment:
I know this line hasn't changed but please extract all magic numbers to constants defined at the top of the file and use them.

Reply:
Please add a FIXME like you did for other magic numbers.